lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -3027,6 +3027,37 @@ LogicalResult ConvertAtenOp<AtenWhereSelfOp>::matchAndRewrite(
   return success();
 }
 
+template <>
+LogicalResult ConvertAtenOp<AtenClampOp>::matchAndRewrite(
+    AtenClampOp op, OpAdaptor adaptor,
+    ConversionPatternRewriter &rewriter) const {
+
+  // Not a tensor type.
+  auto selfType = adaptor.self().getType().dyn_cast<TensorType>();
+  if (!selfType)
+    return rewriter.notifyMatchFailure(
+        op, "Only tensor types input are currently supported");
+
+  int64_t int_min, int_max;
+  if (!matchPattern(op.min(), m_TorchConstantInt(&int_min)))
+    return rewriter.notifyMatchFailure(
+        op, "unimplemented: value `int_min` should be a torch constant int");
+
+  if (!matchPattern(op.max(), m_TorchConstantInt(&int_max)))
+    return rewriter.notifyMatchFailure(
+        op, "unimplemented: value `int_max` should be a torch constant int");
+
+  IntegerAttr min_int = rewriter.getI64IntegerAttr(int_min);
+  IntegerAttr max_int = rewriter.getI64IntegerAttr(int_max);
+  FloatAttr min_fp = rewriter.getF32FloatAttr(float(int_min));
```

Comment:
I wonder is an explicit integral->floating conversion needed here?

Reply:
There is the float(int_min) already there.